src/poggit/timeline/TimeLineEvent.php

```diff
@@ -70,6 +70,6 @@ public function dispatchFor(int $uid) {
 
     public function dispatch(): int {
         return MysqlUtils::query("INSERT INTO event_timeline (type, details) VALUES (?, ?)",
-            "is", $this->getType(), json_encode($this))->insert_id;
+            "is", $this->getType(), json_encode((array) $this))->insert_id;
```

Comment:
Actually I made the jsonSerialize() function to unset `eventId` and `created`, which are already in the database.